ocular/src/chain/registry.rs

```diff
@@ -64,11 +64,21 @@ pub async fn get_chain(name: &str) -> Result<ChainInfo, ChainRegistryError> {
 }
 
 async fn get_content(path: String) -> Result<reqwest::Response, ChainRegistryError> {
-    octocrab::instance()
+    let response = octocrab::instance()
         .repos("cosmos", "chain-registry")
         .raw_file("88bde7fb534ed6f7c26c2073f57ec5135b470f56".to_string(), path)
         .await
-        .map_err(|e| e.into())
+        .unwrap_or_else(|err| {
+            panic!("executor exited with error: {}", err);
```

Comment:
Let's return an error instead,

`ChainRegistryError::Request()` takes in an `octocrab::Error`